config/vufind/Overdrive.ini

```diff
@@ -36,6 +36,10 @@ noAccessString = ""
 ; more overhead with it on. Default is false.
 consortiumSupport = false
 
+;AJAX Status is supported, but you can turn it off it's slowing the search results
```

Comment:
I think this comment is missing an "if" between "off" and "it's." We also by convention include spaces after the `;` in comments.